qutebrowser/browser/shared.py

```diff
@@ -260,3 +262,29 @@ def get_user_stylesheet():
         css += '\nhtml > ::-webkit-scrollbar { width: 0px; height: 0px; }'
 
     return css
+
+
+def netrc_authentication(url, authenticator):
+    """performs authorization using netrc."""
```

Comment:
This should be capitalized and in imperative mood ("Perform ..."), see [PEP 257](https://www.python.org/dev/peps/pep-0257/).

Also, please document the `url` and `authenticator` arguments as well as what the return value means.